### PR TITLE
Makefile: Refer to CONTRIBUTING doc, reword `test` to avoid "example"

### PR DESCRIPTION
**Pull request by EricCousineau-TRI**
## What does this PR do?

Provides concrete example of running a single file, as is shown in other docs.

- [x] Requires #5909 before merging

## Before submitting
- (N/A) Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- (N/A) Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- (N/A) Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [ ] Is this pull request ready for review? (if not, please submit in draft mode)
 - [ ] Check that all items from **Before submitting** are resolved
 - [ ] Make sure the title is self-explanatory and the description concisely explains the PR
 - [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified
 - [ ] **Check that target branch and milestone match!**

## Did you have fun?
Yayyyy! :ski:

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -15,8 +15,8 @@ test:
 	python -m coverage run --source pytorch_lightning -m pytest pytorch_lightning tests pl_examples -v --flake8
 	python -m coverage report
 
-	# specific file
-	# python -m coverage run --source pytorch_lightning -m pytest --flake8 --durations=0 -v -k
+	# example for specific file/test
```
Comment (Borda) [inline Makefile:18]: I think that we do not need it here makefile anymore...
just if it is missing add it to the contribution guide

Comment (EricCousineau-TRI) [reply Makefile]: Done! There's already a mention of how to do this in the section for adding a new test, but I've also added a concrete example for an existing test.

**Comment by Borda** [comment]
@EricCousineau-TRI thank you for sending your PR, just a minor issue coming from our side... we have swapped branches regarding upcoming feat 1.2, mind rebase on actual `master`? if needed see [How to fix PR with mixed base and target branches?](https://pytorch-lightning.readthedocs.io/en/latest/generated/CONTRIBUTING.html#how-to-fix-pr-with-mixed-base-and-target-branches)

**Comment by EricCousineau-TRI** [comment]
Sounds good! I'll wait for #5909 to land before rebasing.

**Comment by Borda** [comment]
> Sounds good! I'll wait for #5909 to land before rebasing.

hi @EricCousineau-TRI your mentioned PR is merged, let's get this done too :]

**Comment by EricCousineau-TRI** [comment]
Done!

**Review by awaelchli — APPROVED** [review]

**Review by Borda — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -205,6 +205,12 @@ This is useful if you do not test against all required dependency versions.
 python -m pytest pytorch_lightning tests pl_examples -v
 ```
 
+You can also run a single test as follows:
+
+```bash
+python -m pytest -v ./tests/trainer/test_trainer_cli.py::test_default_args
```
Comment (carmocca) [inline .github/CONTRIBUTING.md:211]:
```suggestion
python -m pytest -v tests/trainer/test_trainer_cli.py::test_default_args
```

**Review by carmocca — APPROVED** [review]

**Pull request merged**